### ISSUE TITLE
Logging INFO in root logger

### ISSUE THREAD

**Issue by nicaballero** [kislyuk/watchtower#48]
Hi! Glad to meet you. I'm using this package to send all projects logs to Cloudwatch and I'm configuring a root logger to do that:

![image](https://user-images.githubusercontent.com/32133185/33994656-612c2f20-e0a9-11e7-8b6f-79b6700ac460.png)

By other hand, I have the following handler:

![image](https://user-images.githubusercontent.com/32133185/33994731-b02d90e6-e0a9-11e7-933c-4d93114b21bc.png)

With this configuration I'm getting the following error:

![image](https://user-images.githubusercontent.com/32133185/33994823-febd1312-e0a9-11e7-9590-5f9c90097218.png)

To solve this error I must delete the level attribute from my logger, but with that configuration I cannot log any INFO or DEBUG message (maybe because the default value of level is 'WARNING')

Can you help me with this??

Ouch! I forgot to tell you how raise this error :eyes:
To replicate the error you should open a django shell and import logging module. After that, you can write a *logging.info()* or *logging.debug()* sentence

Thank you :)

**Comment by nicaballero** [comment]
Ouch! I forgot to tell you how raise this error :eyes: 
To replicate the error you should open a django shell and import logging module. After that, you can write a *logging.info()* or *logging.debug()* sentence

Thanks again!

**Comment by dvukolov** [comment]
I experienced the same recursion issue when working with the root logger. As soon as you set a log level and attach a CloudWatchLogHandler to the root logger, boto (namely botocore as seen in your traceback) starts logging its own own messages as well. Those in turn are forwarded to the CloudWatchLogHandler which creates a neverending loop. One solution is to attach a filter to the CloudWatchLogHandler like this one:

```
def boto_filter(record):
     return not record.name.startswith('botocore')

handler = CloudWatchLogHandler()
handler.addFilter(boto_filter)
```

**Comment by kislyuk** [comment]
Thanks for this great suggestion. I've implemented and released the filter @dvukolov and @samamorgan . I implemented and released it in v1.0.3.
